taskapp/src/components/home.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import banner from '../banner.jpg';
+
+const HomeComponent = () => {
+    return(
+        <div className="card">
+            <div className="card-header" role="banner">
+                <h2 className="m-4">Welcome to Task Application</h2>
```

Comment:
Lot of hard coded labels, try bring them from an external label file, you should be able to switch labels based on page locale